script/Arcueid/Day 6/06_00_ARC06_1_2.txt

```diff
@@ -344,7 +344,7 @@ It's just that there are a lot of things I can't say.
 [sha:0d7a0d97ebcc271be92f2f697fe57ae0bcd32cd0]{
 -- Page 107, Offset 14184.
 -- 昔から、誠実なんだか不誠実なんだか判断のつかない人でしたから」
-You've always been someone who couldn't make up their mind about whether to be honest or dishonest."
+You've always been someone who couldn't make up his mind about whether to be honest or dishonest."
```

Comment:
Hmm, `his` doesn't click here for me. I feel like it would need to be "**He's** always been someone ... **his** mind". 
I think that the they form works here, alternately "**You've** always been someone ... **your** mind".

Reply:
I think "his" is alright but I'd prefer their